lib/capybara/screenshot/diff/drivers/chunky_png_driver.rb

```diff
@@ -183,10 +184,11 @@ def dimension_changed?(old_image, new_image)
             true
           end
 
-          def draw_rectangles(images, (left, top, right, bottom), (r, g, b))
+          def draw_rectangles(images, (left, top, width, height), (r, g, b))
             images.map do |image|
               new_img = image.dup
-              new_img.rect(left - 1, top - 1, right + 1, bottom + 1, ChunkyPNG::Color.rgb(r, g, b))
+              new_img.rect(left - 1, top - 1, left + width + 1, top + height + 1, ChunkyPNG::Color.rgb(r, g, b))
```

Comment:
@donv yep, sorry for misguiding, ChunkyPNG use `[right, bottom]` style. But JavaScript return rectangle as `[width, height]` style

Reply:
but for `crop` it uses `[width, height]`, hm strange